components/Home/Faq/Faq.tsx

```diff
@@ -1,118 +1,159 @@
-    const swordStates = [generalClicked, beforeClicked, duringClicked];
-    const handleClick = (id: string) => {
-        if (id === "general") {
-            setFaqSectionIndex(0);
-            setGeneralClicked(true);
-            setBeforeClicked(false);
-            setDuringClicked(false);
-        } else if (id === "before") {
-            setFaqSectionIndex(1);
-            setGeneralClicked(false);
-            setBeforeClicked(true);
-            setDuringClicked(false);
-        } else {
-            setFaqSectionIndex(2);
-            setGeneralClicked(false);
-            setBeforeClicked(false);
-            setDuringClicked(true);
+        window.addEventListener('resize', handleResize);
+        
+        return () => {
+            general?.removeEventListener('click', handleGeneralClick);
+            before?.removeEventListener('click', handleBeforeClick);
+            during?.removeEventListener('click', handleDuringClick);
+            window.removeEventListener('resize', handleResize);
         }
-    };
-    return (
+
+    }, [display, mobile]);
+        
+    return ( 
         <section className={styles.faq}>
             <div className={styles.faqContainer}>
-                <div className={styles.faqContainerItem1}>
-                    <Image
-                        className={styles.faqHeader}
-                        alt="faq header"
-                        src={FaqHeader}
-                    />
-                    <Image
-                        className={styles.faqCloud}
-                        alt="cloud"
-                        src={FaqCloud}
-                    />
+                <div className={styles.faqItemContainer1}>
+                    <Image src={FAQHeader} alt="FAQ Header" className={styles.faqHeader} />
                 </div>
-                <div className={styles.faqContainerItem2}>
-                    <div className={styles.faqSwordContainer}>
-                        <div className={styles.faqSwordandHeader}>
-                            <h2>General</h2>
-                            <Image
-                                className={styles.swords}
-                                alt="general sword"
-                                onClick={() => handleClick("general")}
-                                src={generalClicked ? ClickedSword : Sword}
-                            />
-                        </div>
-
-                        <div className={styles.faqSwordandHeader}>
-                            <h2>Before</h2>
-                            <Image
-                                className={styles.swords}
-                                alt="before sword"
-                                onClick={() => handleClick("before")}
-                                src={beforeClicked ? ClickedSword : Sword}
-                            />
-                        </div>
-
-                        <div className={styles.faqSwordandHeader}>
-                            <h2>During</h2>
```

Comment:
Right now all of the buttons are not clicking for the H2 part. You could get around this by also making them a tags with the onclick function and just using CSS to make the text look the same as the h2